CONTRIBUTING.md

```diff
@@ -0,0 +1,80 @@
+## Contributing to the JSON Schema Website
+
+This document outlines how you can get involved and help improve the JSON Schema website. We appreciate all contributions!
+
+## Table of Contents
+
+- [Code of Conduct 📜](#code-of-conduct-📜)
+- [How to Contribute 🚀](#how-to-contribute-🚀)
+    - [Reporting Bugs 🐞](#reporting-bugs-🐞)
+    - [Suggesting Features 💡](#suggesting-features-💡)
+    - [Improving the Website 🌐](#improving-the-website-🌐)
+- [Pull Requests 📬](#pull-requests-📬)
+- [License 📄](#license-📄)
+- [Feedback 💬](#feedback-💬)
+
+##  Code of Conduct 📜
+
+Before contributing, please familiarize yourself with our [Code of Conduct](https://github.com/json-schema-org/.github/blob/main/CODE_OF_CONDUCT.md).
+
+
+##  How to Contribute 🚀
+
+There are several ways you can contribute to the JSON Schema website:
+
+* [**Reporting Bugs** 🐞](#reporting-bugs-🐞)
+* [**Suggesting Features** 💡](#suggesting-features-💡)
+* [**Improving the Website** 🌐](#improving-the-website-🌐)
```

Comment:
Here how's about adding one more section related to docs improvement.

Reply:
Okay, I will do just that. Thanks